tests/zstreamtest.c

```diff
@@ -210,9 +210,9 @@ static int basicUnitTests(U32 seed, double compressibility, ZSTD_customMem custo
     /* context size functions */
```

Comment:
not yet.
The current fuzzer test always reset dictionary to `{NULL,0}` before each new frame, so we can't verify this property.

It's easier to add such test in unit tests.
One way to do it would be 
to make a new compression after a `ZSTD_refPrefix()` one,
and check that decompression works without loading a dictionary.